src/arithmetic/arithmetic_expression_construct.c

```diff
@@ -397,8 +397,9 @@ static AR_ExpNode *_AR_ExpFromMapProjection(const cypher_astnode_t *expr) {
 	const cypher_astnode_t *selector = NULL;
```

Comment:
On line 385 please add an additional example demonstrating the `.*` selection
e.g. ` MATCH (n) RETURN n { .* }`

Reply:
Ok. Example added